modules/customize-certificates-rotate-service-serving.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="rotate-service-serving-{context}"]
+= Manually rotate the generated service certificate
+
+.Prerequisites
+
+* A secret containing the certificate and key pair must 
+have been generated for the service.
+
+.Procedure
+
+. Delete the generated secret for the service:
++
+----
+$ oc delete secret <service.name>
+----
```

Comment:
I'd also add a note after the command that the secret's automatically regenerated under the same name. (Or whatever's really happening here.)

Reply:
I've included a sentence in the description of this command to indicate this is automatically recreated. This is also found in the description and in the following step as well.